src/main/steps/applicant2/enter-your-access-code/content.ts

```diff
@@ -29,7 +29,31 @@ const en = () => ({
 });
 
 // @TODO translations
-const cy = en;
+const cy: typeof en = () => ({
+  title: 'Rhowch eich manylion mynediad',
+  line1: 'Your reference number and access code are in the email you received which invited you to this application.',
```

Comment:
do we not have translations for these bits? should the ticket be blocked in that case?

Reply:
yh im not sure why they didnt as they went through the page and did everything else. i agree with the blocking.

Reply:
note: i did leave a comment in the ticket
https://tools.hmcts.net/jira/browse/NFDIV-2635